site/content/en/docs/api_sdk/cli/_index.md

```diff
@@ -316,3 +321,25 @@ see that command's examples for more information.
   ```bash
   cvat-cli project ls --json > list_of_projects.json
   ```
+
+## Examples - functions
+
+**Note**: The functionality described in this section can only be used
+with the CVAT Enterprise or CVAT Cloud.
+
+- Create a function that uses a detection model from torchvision
+  and run an agent for it:
+
+  ```
+  cvat-cli function create-native "Faster R-CNN" \
+      --function-module cvat_sdk.auto_annotation.functions.torchvision_detection \
+      -p model_name=str:fasterrcnn_resnet50_fpn_v2
```

Comment:
Consider passing arguments as strings by default, this may reduce the number of required type specifications for model parameters.

Reply:
This isn't directly related to this PR, because I'm just reusing the `-p` argument that was implemented for the `task auto-annotate` command.

But for the record, I don't want to do it that way, because it creates opportunities for misinterpretation. E.g. if a shell script runs a command with `-p "$x"`, intending it to be a string parameter, it may unexpectedly be parsed as a different type instead (depending on the value).